<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QToolButton" name="home_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="homeButBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>171</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="settingBut">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>参数设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="videoBut">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>视频播放</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="groundModeBut">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>地面模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="offRoadModeBut">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>越野模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="flyModeBut">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>飞行模式</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="switchPixLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>90</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="choseWidget" native="true">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>400</width>
     <height>340</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>340</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="choseBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextBrowser" name="choseBrowser">
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>290</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="choseLabel">
      <property name="text">
       <string>请选择连接目标：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="searchRobotBut">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>检测可连接设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectRobotBut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>连接设备</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groundInformation">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>236</width>
     <height>410</height>
    </rect>
   </property>
   <property name="title">
    <string>参数信息</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="groundMap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>200</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid dimgray;
</string>
    </property>
    <property name="text">
     <string>速度：- -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lngLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid dimgray;
</string>
    </property>
    <property name="text">
     <string>经度：- -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="latLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid dimgray;
</string>
    </property>
    <property name="text">
     <string>纬度：- -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="flyInformation">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>236</width>
     <height>520</height>
    </rect>
   </property>
   <property name="title">
    <string>参数信息</string>
   </property>
   <widget class="QLabel" name="vxLabel_fly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid dimgray;
</string>
    </property>
    <property name="text">
     <string>水平速度：- -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vyLabel_fly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid dimgray;
</string>
    </property>
    <property name="text">
     <string>垂直速度：- -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lngLabel_fly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid dimgray;
</string>
    </property>
    <property name="text">
     <string>经度：- -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="latLabel_fly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid dimgray;
</string>
    </property>
    <property name="text">
     <string>纬度：- -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="heigh_fly">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid dimgray;
</string>
    </property>
    <property name="text">
     <string>高度：- -</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="WidgetPFD" name="posturePanel" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="flyModeFunBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>322</width>
     <height>96</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>96</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>96</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="fly_Home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>26</y>
      <width>3</width>
      <height>64</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="fly_qifei">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>26</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>26</y>
      <width>3</width>
      <height>64</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="fly_jiangluo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>26</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="fly_xuanting">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>26</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>26</y>
      <width>3</width>
      <height>64</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="videoLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>660</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeVideoBut">
   <property name="geometry">
    <rect>
     <x>725</x>
     <y>40</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 40px;
    color: rgb(71, 62, 62);
	padding-top:-5px;

}
QPushButton:hover {
background-color: rgb(255, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="jsTestBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>201</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>JS测试</string>
   </property>
   <widget class="QLabel" name="labelLX">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>LeftX:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LXValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelLY">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>LeftY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LYValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RXValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelRX">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RightX:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RYValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelRY">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RightY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLS1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>LeftS1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LS1Value">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelLS2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>LeftS2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LS2Value">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelRS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RightS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RSValue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>flyModeFunBox</zorder>
  <zorder>home_Button</zorder>
  <zorder>switchPixLabel</zorder>
  <zorder>choseWidget</zorder>
  <zorder>searchRobotBut</zorder>
  <zorder>connectRobotBut</zorder>
  <zorder>groundInformation</zorder>
  <zorder>flyInformation</zorder>
  <zorder>videoLabel</zorder>
  <zorder>homeButBox</zorder>
  <zorder>closeVideoBut</zorder>
  <zorder>jsTestBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPFD</class>
   <extends>QWidget</extends>
   <header>WidgetPFD.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
